gradlew build throws up : NoClassDefFoundError: org/slf4j/impl/StaticLoggerBinder

When i try `.gradlew run` as per README:
i am hitting:
```
Starting a new Gradle Daemon for this build (subsequent builds will be faster).
[buildinfo] Not using buildInfo properties file for this build.
:cassandra:compileJava UP-TO-DATE
:cassandra:compileScala UP-TO-DATE
:cassandra:processResources UP-TO-DATE
:cassandra:classes UP-TO-DATE
:cassandra:run
Exception in thread "main" java.lang.NoClassDefFoundError: org/slf4j/impl/StaticLoggerBinder
	at org.apache.spark.Logging$class.initializeLogging(Logging.scala:116)
	at org.apache.spark.Logging$class.initializeIfNecessary(Logging.scala:107)
	at org.apache.spark.Logging$class.log(Logging.scala:51)
	at org.apache.spark.SparkContext.log(SparkContext.scala:79)
	at org.apache.spark.Logging$class.logInfo(Logging.scala:59)
	at org.apache.spark.SparkContext.logInfo(SparkContext.scala:79)
	at org.apache.spark.SparkContext.<init>(SparkContext.scala:205)
	at org.openzipkin.dependencies.spark.cassandra.ZipkinDependenciesJob$.run(ZipkinDependenciesJob.scala:84)
	at org.openzipkin.dependencies.spark.cassandra.ZipkinDependenciesJob$.main(ZipkinDependenciesJob.scala:71)
	at org.openzipkin.dependencies.spark.cassandra.ZipkinDependenciesJob.main(ZipkinDependenciesJob.scala)
Caused by: java.lang.ClassNotFoundException: org.slf4j.impl.StaticLoggerBinder
	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	... 10 more
:cassandra:run FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':cassandra:run'.
> Process 'command '/usr/lib/jvm/java-7-oracle-1.7.0.45/bin/java'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 15.727 secs
```

There are other classpath problems. For example shadowJar doesn't
quite work yet:

$ ./gradlew shadowJar
$ java -jar ./cassandra/build/libs/cassandra-1.21.1-SNAPSHOT-all.jar
--snip--
Caused by: java.lang.ClassNotFoundException: org.apache.spark.SparkContext

Needs a little more work I think.

--- a/build.gradle
+++ b/build.gradle
@@ -247,7 +247,6 @@ subprojects { subproject ->
         testCompile 'junit:junit:4.12'
         testCompile 'org.mockito:mockito-all:1.10.19'
         testCompile "org.scalatest:scalatest_${scalaInterfaceVersion}:2.2.5"
-        testRuntime "org.slf4j:slf4j-simple:${commonVersions.slf4j}"
     }
 }
 

--- a/cassandra/build.gradle
+++ b/cassandra/build.gradle
@@ -1,7 +1,11 @@
 apply plugin: 'application'
 apply plugin: 'com.github.johnrengelman.shadow'
 mainClassName = 'org.openzipkin.dependencies.spark.cassandra.ZipkinDependenciesJob'
-test.maxParallelForks = 1
+
+test {
+    maxParallelForks 1
+    systemProperty 'log4j.configuration', 'org/apache/spark/log4j-defaults.properties'
+}
 
 ext {
     versions = [

--- a/gradle/dependencies.gradle
+++ b/gradle/dependencies.gradle
@@ -37,6 +37,6 @@ subprojects { subproject ->
         // Log4j is bound by Spark
         exclude group: "ch.qos.logback"
         // don't loop slf4j-log4j-slf4j
-        exclude module: "slf4j-log4j12"
+        exclude module: "log4j-over-slf4j"
      }
 }

